File: scala-example/spring-boot-example/src/main/scala/com/github/poad/scala/springboot/example/content/Message.scala
package com.github.poad.scala.springboot.example.content

import javax.xml.bind.annotation.{XmlElement, XmlRootElement}

import scala.beans.BeanProperty

@XmlRootElement
case class Message(@BeanProperty @XmlElement id: String, @BeanProperty @XmlElement message: String) {

}

File: java-example/spring-boot/spring-boot-web-example/src/main/java/com/github/poad/examples/java/webapps/api/Message.java
package com.github.poad.examples.java.webapps.api;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class Message {
    @XmlElement
    private String id;
    @XmlElement
    private String message;

    public Message() {
        this(null, null);
    }

    public Message(String id, String message) {
        this.id = id;
        this.message = message;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}
